schema.sql

```diff
@@ -9,10 +9,20 @@ create table ballots (
   batch_id varchar(36),
   original_filename text unique,
   normalized_filename text unique,
+
+  -- Original interpretation of the page. This value should never be updated.
+  -- @type {PageInterpretation}
   interpretation_json text not null,
+
+  -- Changes made in adjudication, should be applied on top of the original CVR.
+  -- Updated as the page is adjudicated.
+  -- @type {MarksByContestId}
   adjudication_json text,
-  adjudication_info_json text,
+
+  -- Does this page _currently_ require adjudication? Updated as the page is
+  -- adjudicated.
```

Comment:
these are great comments.

one alternative to consider:

```
requires_adjudication boolean,
finished_adjudication_at datetime,
```

That way you can always go back and see which ones needed adjudication at some point, and then were adjudicated.

Reply:
I like this idea. Will update.